ajax/libs/stretchy/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "stretchy",
+  "version": "1.1.0",
+  "description": "Form element autosizing, the way it should be.",
+  "filename": "stretchy.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LeaVerou/stretchy.git"
+  },
+  "author": "Lea Verou",
+  "license": "MIT",
+  "keywords": [
+    "element autosizing"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/LeaVerou/stretchy.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "*.scss",
+          "stretchy*",
+          "style*"
```

Comment:
I think `style*` are for `index.html`. I think we just need `"stretchy*"` here. Thanks.